src/std/make.ss

```diff
@@ -95,7 +95,10 @@ TODO:
 (defstruct settings
   (srcdir libdir bindir prefix force optimize debug static-debug verbose build-deps
    libdir-prefix parallelize
-   full-program-optimization)
+   full-program-optimization
+   build-release
+   build-optimized
+   )
```

Comment:
put the paren right after the last field?

Reply:
fixed.